testingii/src/App.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+// import Dashboard from './dashboard/Dashboard.js';
+import { render, fireEvent } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+const { getByText } = render(<App />);
+
+it('adds a strike', () => {
```

Comment:
There are some issues with this test.  Or, at the very least, there should be instances where this test fails because your strike function has bugs.